.travis.yml

```diff
@@ -0,0 +1,37 @@
+language: go
+sudo: required
+go:
+- 1.8
+- 1.9
+- tip
+go_import_path: github.com/kata-containers/ksm-throttler
+
+before_install:
+- go get github.com/mattn/goveralls
```

Comment:
Just an idea, but how about trying out codecov.io instead? https://github.com/clearcontainers/jenkins/issues/28 Or we could run both and see how they compare?

Reply:
Added